Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -18,14 +18,7 @@ class PetSingleDetailsViewController: UIViewController {
     @IBOutlet weak private var saveSinglePetButton: UIButton!
 
     // MARK: - Vars/Lets
-    private var singlePet: AdoptPet?
-    private var indexSinglePet: Int = 0
-    private var namePet = ""
-    private var breedPet = ""
-    private var genderPet = ""
-    private var agePet = ""
-    private var imgPet = ""
-    private let viewContext = (UIApplication.shared.delegate as? AppDelegate)?.persistentContainer.viewContext
+    private lazy var singlePetViewModel = SinglePetViewModel(repository: SinglePetRepository())
```

Comment:
I removed the instance variablesRemoved all the instance variables that were inside the PetSingleViewController